--- a/ltmain.sh
+++ b/ltmain.sh
@@ -7470,8 +7470,8 @@
       # Tru64 UNIX uses -model [arg] to determine the layout of C++
       # classes, name mangling, and exception handling.
       # Darwin uses the -arch flag to determine output architecture.
       # -q <option> for IBM XL C/C++ compiler.
-      -model|-arch|-isysroot|--sysroot|-q)
+      -model|-arch|-isysroot|--sysroot|-q|-target)
 	func_append compiler_flags " $arg"
 	func_append compile_command " $arg"
 	func_append finalize_command " $arg"
